.github/workflows/pull-request.yaml

```diff
@@ -12,6 +12,9 @@ jobs:
       CI_TOOLS_URL: https://github.com/samuelattwood/partner-charts-ci/releases/latest/download/partner-charts-ci-linux-amd64
 
     steps:
+      - uses: actions/setup-node@v2
+        with:
+           node-version: '16.x'
```

Comment:
Why not `node-version: 18`?

Reply:
version 16 is sufficient enough and I didn't want to make a huge leap in versions.

Reply:
Node 16 is EOL. https://endoflife.date/nodejs